x/curating/types/errors.go

```diff
@@ -6,9 +6,9 @@ import (
 
 // x/curating module sentinel errors
 var (
-	ErrPostNotFound  = sdkerrors.Register(ModuleName, 1, "Post not found")
-	ErrAlreadyVoted  = sdkerrors.Register(ModuleName, 2, "Already voted")
-	ErrDuplicatePost = sdkerrors.Register(ModuleName, 3, "Post already exists")
-	ErrPostExpired   = sdkerrors.Register(ModuleName, 4, "Post already expired")
-	ErrInvalidPostID = sdkerrors.Register(ModuleName, 5, "PostID cannot be nil")
+	ErrPostNotFound  = sdkerrors.Register(ModuleName, 2, "Post not found")
```

Comment:
Maybe worth adding a comment why these are starting with 2 instead of 1?

Reply:
good idea will add a note